Solutions/GoogleCloudPlatformIDS/Data Connectors/GCPCloudIDSLog_CCP/GCPCloudIDSLog_DCR.json

```diff
@@ -0,0 +1,77 @@
+[
+    {
+        "name": "GCPCLOUDIDSDCR",
+        "apiVersion": "2021-09-01-preview",
+        "type": "Microsoft.Insights/dataCollectionRules",
+        "location": "{{location}}",
+        "properties": {
+            "dataCollectionEndpointId": "{{dataCollectionEndpointId}}",
+            "streamDeclarations": {
+                "Custom-GCPCLOUDIDS": {
+                    "columns": [
+                        {
+                            "name": "insertId",
+                            "type": "string"
+                        },
+                        {
+                            "name": "logName",
+                            "type": "string"
+                        },
+                        {
+                            "name": "protoPayload",
+                            "type": "dynamic"
+                        },
+                        {
+                            "name": "resource",
+                            "type": "dynamic"
+                        },
+                        {
+                            "name": "severity",
+                            "type": "string"
+                        },
+                        {
+                            "name": "timestamp",
+                            "type": "datetime"
+                        },
+                        {
+                            "name": "jsonPayload",
+                            "type": "dynamic"
+                        },
+                        {
+                            "name": "receiveTimestamp",
+                            "type": "datetime"
+                        },
+                        {
+                            "name": "operation",
+                            "type": "string"
+                        },
+                        {
+                            "name": "TimeGenerated",
```

Comment:
Are we getting the TimeGenerated field as part of the input stream, if not, please remove.